en/docs/quick-starts/auth-code-playground.md

```diff
@@ -61,21 +61,19 @@ This page guides you through using a **sample Playground application** to try ou
 
 4. Provide the requested consent and enter the following details on the screen that appears. 
 
-    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+    - **Callback URL:** `http://wso2is.local:8080/playground2/oauth2client`
 
-    - **Access Token Endpoint:** https://localhost:9443/oauth2/token 
+    - **Access Token Endpoint:** `https://localhost:9443/oauth2/token` 
```

Comment:
`<IS_HOST>:<IS_PORT>`

Reply:
Leaving it with the values since it's a sample.